lib/user_agent/browsers/apple_core_media.rb

```diff
@@ -29,27 +29,21 @@ def security
       end
 
       def os
-        if application
-          if application.comment[0] =~ /Windows NT/
-            OperatingSystems.normalize_os(application.comment[0])
-          elsif application.comment[2].nil?
-            OperatingSystems.normalize_os(application.comment[1])
-          elsif application.comment[1] =~ /Android/
-            OperatingSystems.normalize_os(application.comment[1])
-          else
-            OperatingSystems.normalize_os(application.comment[2])
-          end
+        return unless application
+
+        if application.comment[0] =~ /Windows NT/
+          OperatingSystems.normalize_os(application.comment[0])
+        elsif application.comment[2].nil?
+          OperatingSystems.normalize_os(application.comment[1])
+        elsif application.comment[1] =~ /Android/
+          OperatingSystems.normalize_os(application.comment[1])
         else
-          nil
+          OperatingSystems.normalize_os(application.comment[2])
         end
       end
 
       def localization
-        if application.nil?
-          nil
-        else
-          application.comment[3]
-        end
+        application.comment[3] if application
```

Comment:
I made the comment because I paused for a second before I realised it that it was logically the same as `return unless application` in the other methods. I wouldn't have had to think about it at all if they were visually the same too so I thought it worth mentioning as I always favour clarity over elegance.

If you'd prefer to leave them as they are that's ok, it's a very minor nitpick really.